packages/flutter_tools/gradle/src/main/kotlin/dependency_version_checker.gradle.kts

```diff
@@ -104,6 +104,7 @@ class DependencyVersionChecker {
          * we treat it as within the range for the purpose of this check.
          */
         fun checkDependencyVersions(project: Project) {
+            project.extra.set("failedDependencyChecks", false)
```

Comment:
Please make the string a constant in this file and define what the variable means and when it should be set.